### ISSUE TITLE
Access github: Your connection is not secure

### ISSUE THREAD

**Issue by kuchaguangjie** [XX-net/XX-Net#4126]
**Issue:**

When access github from firefox, get error tip:

> github.com uses an invalid security certificate.
> The certificate expired on 08/28/2016 11:45 PM. The current time is 08/29/2016 ...
> Error code: SEC_ERROR_EXPIRED_CERTIFICATE

There are similar issue for https://www.google.com.hk/, but https://www.google.com/ works fine.
Not sure is the certification expired, or some other issue.

**Screenshot:**
![github - issue](https://cloud.githubusercontent.com/assets/4230959/18036119/f9697dfc-6d95-11e6-87f4-3df16e75f6af.png)

**Comment by jzp820927** [comment]
好像是证书问题，但是你说 只有 https://www.google.com.hk/ 有问题，https://www.google.com/ 又没问题，那你就用 https://www.google.com/ 嘛

**Comment by cainiaowu** [comment]
Deleting Cert8.db for Firefox, and all site specific cert regenerated by xxnet (data\gae_proxy\certs).

**Comment by kuchaguangjie** [comment]
@cainiaowu I modified the source code `xxnet/code/default/gae_proxy/local/cert_util.py` a little bit, to make the default validity as 10 years, and it's configurable via a static variable `ca_validity` from class `CertUtil`. And created a pull request, maybe you can check that to see is it reasonable & acceptable, thanks.

refer: https://github.com/XX-net/XX-Net/pull/4130/commits/ba15b5e957c6bf3253e0586d5458ab5b4584cfbb

**Comment by cainiaowu** [comment]
But I am not the developer. @yeahwu
https://github.com/XX-net/XX-Net/graphs/contributors

**Comment by chinauser2013** [comment]
I've been having this problem on Chrome for a long time.  I just noticed if I try to import the CA.crt file it doesn't show up in the Trusted Root Certificates.  This just happens on one of my laptops but not the other.  (Also on the computer that it's not working on, I have this problem if I'm using Chrome or IE but it works if I use Edge.)  Any suggestions what I should do?

**Comment by kuchaguangjie** [comment]
@chinauser2013 I already gave the steps to fix it, you can following those steps and try to fix it first.

Also I have fixed it in source code, and created a pull request, but seems the maintainer didn't response yet.

**Comment by chinauser2013** [comment]
Yes I tried those steps but still have the same problem.  

I have imported the certificate.  It says "Import Successful" but when I go into to see if it's in the Trusted Root Certification Authorities, I don't see a GoAgent CA or GoAgent XX-Net there.  That's on the computer where XX-Net is not working (let's called in Computer A), 

On the computer where XX-Net is working (Computer B), I can see the GoAgent CA and GoAgent XX-Net in the Trust Root Certification Authorities.

That leads me to believe something on Computer A is blocking the certficate from being installed but I don't know what it is.  I did have an Antivirus program on there before, Kaspersky and uninstalled it but it shows some Kaspersky certificates still there.  I wonder if that's the problem?
